homework5.js

```diff
@@ -0,0 +1,58 @@
+$('document').ready(function() {
+	console.log('this is running');
+});
+
+$('#submit-btn').click(function(event) {
+	event.preventDefault()
+	var city = $('#city-type').val();
+	city = (city.toLowerCase());
+
+	if (city === 'nyc'|| city === 'new york'||city === 'ny' || city === 'new york city') {
+	$('body').removeClass('nyc sf la austin sydney').addClass('nyc');
+}
+});
+
+$('#submit-btn').click(function(event) {
```

Comment:
You only needed to declare this once and then follow up with a series of if/else statements.